### PR TITLE
MLPerf: add flagfiles, remove eval games against target 

### PR DESCRIPTION
**Pull request by tommadams**
Eval games against target will be played out-of-band at the end of the training run.

### CODE REVIEW

File: ml_perf/flags/9/train.flags [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Flags for training & validation.
+
+--conv_width=64
+--fc_width=64
+--trunk_layers=10
```
Comment (amj) [inline ml_perf/flags/9/train.flags]: trunk_layers=9

File: ml_perf/reference_implementation.py [hunk 2]
```diff
@@ -225,28 +244,24 @@ def rl_loop():
     #     generations.
     #   - uniformly resample the window each iteration (see TODO in selfplay
     #     for more info).
-    tf_records = get_golden_chunk_records(min(5, state.iter_num));
+    tf_records = get_golden_chunk_records(min(5, state.iter_num))
```
Comment (amj) [inline ml_perf/reference_implementation.py]: can we extract this to a flag?  `window_size` or similar.

**Comment by tommadams** [comment]
Addressed comments. I also changed `gating_win_rate` to `0.5` as we discussed yesterday.

**Comment by instinct79** [comment]
Are the files for flags/19/ put as a placeholder for now ? flags/19/train.flags shows conv_width=64 while dual_net.py has that set to 256 for 19x19.

-Nitin.

**Comment by amj** [comment]
yes, the flags/19 are a placeholder, sorry!

**Pull request merged**